packages/node_modules/@webex/plugin-meetings/src/meetings/util.js

```diff
@@ -22,6 +24,15 @@ MeetingsUtil.extractDestination = (destination, type) => {
   return dest;
 };
 
+// ** This is a workaround to fix Firefox and google android not downloading H264 codec on start
+MeetingsUtil.triggerH264Codec = async () => {
```

Comment:
This feature should be in its own commit or PR as it does not align with `fix(meeting-plugin): fix media ready event not triggering multiple times`